v7/static_comments/README.md

```diff
@@ -0,0 +1,66 @@
+This plugin allows to add static comments to your theme. Static comments are taken from files `<path>/<name>.<id>.wpcomment`, where `<path>/<name>.<ext>` is the post's main file name. Such comments are for example written by the `import_wordpress` plugin when specifying the `--export-comments` argument on import.
+
+You must use a theme which supports static comments for them to be visible (see below for instructions on how to adjust a theme).
+
+
+Why use static comments?
+------------------------
+
+Static comments allow you to avoid using a dynamic (JavaScript-based) comment system. If you want users to be able to still comment things, you need to provide a form which could send the comment as an email to you, so you can create the comment files manually (or with a script).
+
+Static comments also allow you to import a legacy WordPress blog and convert it to a completely static Nikola blog, without having to use some external service for handling the comments.
+
+
+Comment files
+-------------
+
+Comment files are of the following form::
+
+    .. id: 10
+    .. approved: True
+    .. author: felix
+    .. author_email: felix@fontein.de
+    .. author_IP: 1.2.3.4
+    .. author_url: https://felix.fontein.de
+    .. date_utc: 2017-01-06 11:23:55
+    .. parent_id: 8
+    .. wordpress_user_id: 1
+    .. compiler: rest
+
+    this is a test comment.
+
+    the content spans the rest of the file.
+
+Most header fields are optional. `compiler` must specify a page compiler which allows to compile a content given as a string to a string; these are currently the restructured text compiler (`rest`), the `ipynb` compiler, and the [WordPress](https://plugins.getnikola.com/#wordpress_compiler) (`wordpress`) compiler. Comments can form a hierarchy; `parent_id` must be the comment ID of the parent comment, or left away if there's no parent.
```

Comment:
Well, it should be up to the blog admin to decide which page compiler he/she whats to use for which comments. If for whatever reason they want to use `ipynb`, they can do so because that plugin supports compiling from a string to a string.

I'll add passthrough support for `html`.

I can also add such a function to the `markdown` plugin in Nikola core to enable using `markdown` as a comment compiler.